src/redux/reducers/index.js

```diff
@@ -38,8 +38,8 @@ const createCollection = (label = 'Untitled Collection') => {
     created: timeCreated,
     lastUpdated: timeCreated,
     label,
-    selectedResourceType: null,
-    resourceTypeFilters: Object.keys(PLURAL_RESOURCE_TYPES)
+    selectedResourceType: TYPES_SORTED_BY_LABEL[0],
+    resourceTypeFilters: TYPES_SORTED_BY_LABEL
```

Comment:
I realize that Master does the same, but it seems pretty fake the way we establish the initial `resourceTypeFilters` through a list of constants.  It should be dynamic and built based on the records returned for the patient. 

Is it confirmed that all the test patients have at least 1 resource for each resourceType?

Reply:
The type filters don't respond to what records are available. This seems like something that could be adapted, here,  _after_ that functionality is implemented.